ospfd/ospf_vty.c

```diff
@@ -6368,9 +6818,12 @@ DEFUN (no_ip_ospf_dead_interval,
   /* Update timer values in neighbor structure. */
   if (argc == 1)
     {
-      struct ospf *ospf;
-      
-      if ((ospf = ospf_lookup()))
+      struct ospf *ospf = ospf_lookup_by_vrf_id (ifp->vrf_id);
+
+      if (IS_DEBUG_OSPF_TRACE)
```

Comment:
I have removed it in latest patch. The debug_trace is meant to use for internal daemon activity/probing logs. function name printed in logs provides which cli is executed with parameters.